app/src/main/java/org/groundplatform/android/ui/datacollection/components/ButtonAction.kt

```diff
@@ -33,10 +33,15 @@ enum class ButtonAction(
   PREVIOUS(Theme.TRANSPARENT, textId = R.string.previous),
   SKIP(Theme.TRANSPARENT, textId = R.string.skip),
   UNDO(
-    Theme.LIGHT_GREEN,
+    Theme.TRANSPARENT,
```

Comment:
Why was this changed to transparent? In the mocks it appears to be light green.

Reply:
Followed the new one, as with older one, big green buttons are not looking great.


<img width="530" alt="Screenshot 2025-05-21 at 9 46 52 PM" src="https://github.com/user-attachments/assets/b1fd7c20-5d76-4caa-a0cc-14036d5f04c1" />